reporter-web-app/config/env.js

```diff
@@ -42,12 +42,12 @@ dotenvFiles.forEach(dotenvFile => {
 
 // We support resolving modules according to `NODE_PATH`.
 // This lets you use absolute paths in imports inside large monorepos:
-// https://github.com/facebookincubator/create-react-app/issues/253.
+// https://github.com/facebook/create-react-app/issues/253.
```

Comment:
Commit message: "Change based dependencies", I don't get that sentence. Are you missing an "on"?

Reply:
Env.js comes directly from create-react-app - will need to fill a PR with upstream to get this fixed